src/libraries/System.Private.CoreLib/src/System/Range.cs

```diff
@@ -77,6 +87,23 @@ public override string ToString()
             pos += charsWritten;
 
             return new string(span.Slice(0, pos));
+#else
+            var builder = new StringBuilder(2 + (2 * 11)); // 2 for "..", then for each index 1 for '^' and 10 for longest possible uint
+            if (Start.IsFromEnd)
+            {
+                builder.Append('^');
+            }
+            builder.Append((uint)Start.Value);
+            builder.Append('.');
+            builder.Append('.');
+            if (End.IsFromEnd)
+            {
+                builder.Append('^');
+            }
+            builder.Append((uint)End.Value);
+            return builder.ToString();
```

Comment:
Similarly here, I'd suggest just keeping it simple:
```C#
return Start.ToString() + ".." + End.ToString();
```
In this case, it will result in more allocation, but it's also probably not worth the complexity to try to avoid.